fineract-provider/build.gradle

```diff
@@ -444,13 +442,10 @@ tasks.withType(JavaCompile) {
                 "DoubleBraceInitialization",
                 "InconsistentCapitalization",
                 "MissingOverride",
-//                "ReturnMissingNullable",
 //                "InconsistentOverloads",
 //                "MethodCanBeStatic",
 //                "Var",
 //                "ConstantField",
-//                "ConstructorInvokesOverridable",
```

Comment:
see https://errorprone.info/bugpattern/ConstructorInvokesOverridable .. how many of this are there, is this hard to fix? I think you may be able to easily fix it just by making something methods that are called from constructors `private` and/or `final`...

Reply:
same as ReturnMissingNullable

Reply:
it fails with ConstructorInvokesOverridable is not a valid checker name